readme.md

```diff
@@ -6,6 +6,8 @@ Configure Caddy sites role.
 ## Role Variables
 
 - `caddy_sites_reverse_proxy`: A list of sites, that should be configured as reverse proxy.
+- `caddy_sites_redirect`: A list of sites, that should be configured a redirect.
+- `caddy_sites_custom`: A list of sites, that should a custom configuration.
```

Comment:
I think this sentence is incomplete. Should this be used for sites that have their own certificates?